lib/kitchen/directions/bake_learning_objectives.rb

```diff
@@ -9,9 +9,13 @@ def self.v1(chapter:)
         end
       end
 
-      def self.v2(chapter:, add_title: true)
+      def self.v2(chapter:, add_title: true, li_numbering: false)
```

Comment:
or, actually, could it have a default symbol value like `:in_chapter` to indicate what the default option does?

Reply:
@kswanson33 I chose nil, because if we will change the value to something else we need to update every recipe which utilizes this version and remember to add it in a new one if we will be using this version in any book. But let me know what do you think?

Reply:
`:in_chapter` could be set to the default value for this direction, just like how `nil` is being set as the default value now, and recipes still wouldn't have to be updated :)

Reply:
Thank you Kendra, I didn't think of it :slightly_smiling_face: